src/index.ts

```diff
@@ -223,6 +223,8 @@ export function toBns(n: number | string): string {
     out = sciNotation(base, exponent)
   }
 
+  validate(out)
```

Comment:
This is redundant for most operations since they already call validate. Create a toBnsInner that gets used by other ops.